File: [CODE]/rightoversBlazorNWEB/Domain/DTOs/UserLoginDto.cs
namespace Domain.DTOs;

public class UserLoginDto
{

    public UserLoginDto(string userName, string password)
    {
        UserName = userName;
        Password = password;
    }

    public UserLoginDto()
    {
    }

    public string UserName { get; set; }
    public string Password { get; set; }
}

File: [CODE]/rightoversJAVA/src/main/java/sep3/g3/rightoversjava/model/UserLoginDTO.java
package sep3.g3.rightoversjava.model;

public class UserLoginDTO {
    private String username;
    private String password;

    public UserLoginDTO(String username, String password) {
        this.username = username;
        this.password = password;
    }

    public String getUsername() {
        return username;
    }

    public String getPassword() {
        return password;
    }
}
